core/src/main/kotlin/ch/ergon/dope/QueryBuilder.kt

```diff
@@ -22,7 +23,13 @@ class QueryBuilder {
 
     fun selectFrom(fromable: Fromable) = SelectClause(AsteriskExpression()).from(fromable)
 
+    fun selectFrom(bucket: AliasedBucket) = SelectClause(AsteriskExpression()).from(bucket)
```

Comment:
name the argument aliasedBucket, otherwise there will be confusion since there exists a type Bucket as well

Reply:
rename it everywhere